src/scrabble.js

```diff
@@ -1,5 +1,34 @@
-function scrabble() {
+const points = require('./points')
+const { letterBonusChecker, wordBonusChecker } = require('./bonusCheckers')
 
+function scrabble(word) {
+  // guard clause to root out the null and empty string values
+  if (!word || word.trim() === '') return 0
+
+  // force the word to lower case and then split into an array for easier looping
+  const wordArray = word.toLowerCase().split('')
+  let score = 0
+
+  // forEach version, nicer looking and easier to understand
```

Comment:
Really like the clean, efficient code alongside clear comments, shows exact intent of the code.